builder/builder.go

```diff
@@ -104,7 +104,7 @@ type Payload struct {
 }
 
 func (b *Builder) Build(ctx context.Context, payload *Payload) (*monomer.Block, error) {
-	txs := slices.Clone(payload.InjectedTransactions) // Shallow clone is ok, we just don't want to modify the slice itself.
+	cosmosTxs := slices.Clone(payload.InjectedTransactions) // Shallow clone is ok, we just don't want to modify the slice itself.
```

Comment:
It's generally best to avoid renaming unnecessarily. It creates a large diff that's difficult to review.